pkg/bundle/internal/target/target.go

```diff
@@ -354,31 +367,31 @@ func prepareTargetPatch[T targetApplyConfiguration[T]](target T, bundle trustapi
 		)
 }
 
-type targetData struct {
-	data       string
-	binaryData map[string][]byte
+type Data struct {
+	Data       string
+	BinaryData map[string][]byte
 }
```

Comment:
comment: Looking at this now, I think we could use better names. Obviously it's not related to this PR, but `Data.PEM` and `Data.Binary` or something would be more descriptive and avoid echoing `Data.Data` 🤔

Reply:
Agreed! I can try to fix this in one of my follow-up PRs.

Reply:
Ah, the data struct is unlike the non-data struct. 😄